replicate/base_model.py

```diff
@@ -4,7 +4,10 @@
     from replicate.client import Client
     from replicate.collection import Collection
 
-import pydantic
+try:
+    from pydantic import v1 as pydantic
+except ImportError:
+    import pydantic
```

Comment:
When Pydantic 1 is installed, Mypy generates the following error:

> replicate/base_model.py:8: error: Module "pydantic" has no attribute "v1"  [attr-defined]


When Pydantic 2 is installed, Mypy generates the following error:

> replicate/base_model.py:10: error: Name "pydantic" already defined (by an import)  [no-redef]

There's probably a better way to solve this, but adding `# type: ignore` resolved the issues for me.

```suggestion
try:
    from pydantic import v1 as pydantic  # type: ignore
except ImportError:
    import pydantic  # type: ignore
```